File: Python/Warmup-1/sum_double.py
"""
Retorne a soma dos dois números dados, exceto quando ele forem iguais, nesse caso
retorne o dobro da soma.
"""


def sum_double(a, b):
    if a != b:
        return a + b
    else:
        return 2 * (a + b)


print(sum_double(12, 4))

File: Java/Warmup-1/sumDouble/SumDouble.java
package sumDouble;

public class SumDouble {
    //Retorne a soma de dois valores, a menos que eles sejam iguais. Desse modo, retorne o dobro da soma.

    public int sumDouble(int a, int b) {
        if (a != b){
            return (a+b);
        } else return 2*(a+b);
    }
}
